packages/vaadin-lumo-styles/src/components/tabs.css

```
@@ -0,0 +1,284 @@
+}
+
+[part='forward-button']::after {
+  content: var(--lumo-icons-angle-left);
+}
+
+:host([orientation='vertical']) [part='back-button'],
+:host([orientation='vertical']) [part='forward-button'] {
+  display: none;
+}
+
+/* Tabs overflow */
+
+[part='tabs'] {
+  --_lumo-tabs-overflow-mask-image: none;
+  -webkit-mask-image: var(--_lumo-tabs-overflow-mask-image);
+  mask-image: var(--_lumo-tabs-overflow-mask-image);
+}
+
+/* Horizontal tabs overflow */
+
+/* Both ends overflowing */
+:host([overflow~='start'][overflow~='end']:not([orientation='vertical'])) [part='tabs'] {
+  --_lumo-tabs-overflow-mask-image: linear-gradient(
+    90deg,
+    transparent 2em,
+    #000 4em,
+    #000 calc(100% - 4em),
+    transparent calc(100% - 2em)
+  );
+}
+
+/* End overflowing */
+:host([overflow~='end']:not([orientation='vertical'])) [part='tabs'] {
+  --_lumo-tabs-overflow-mask-image: linear-gradient(90deg, #000 calc(100% - 4em), transparent calc(100% - 2em));
+}
+
+/* Start overflowing */
+:host([overflow~='start']:not([orientation='vertical'])) [part='tabs'] {
+  --_lumo-tabs-overflow-mask-image: linear-gradient(90deg, transparent 2em, #000 4em);
+}
+
+/* Vertical tabs overflow */
+
+/* Both ends overflowing */
+:host([overflow~='start'][overflow~='end'][orientation='vertical']) [part='tabs'] {
+  --_lumo-tabs-overflow-mask-image: linear-gradient(transparent, #000 2em, #000 calc(100% - 2em), transparent);
+}
+
+/* End overflowing */
+:host([overflow~='end'][orientation='vertical']) [part='tabs'] {
+  --_lumo-tabs-overflow-mask-image: linear-gradient(#000 calc(100% - 2em), transparent);
+}
+
+/* Start overflowing */
+:host([overflow~='start'][orientation='vertical']) [part='tabs'] {
+  --_lumo-tabs-overflow-mask-image: linear-gradient(transparent, #000 2em);
+}
+
+:host [part='tabs'] ::slotted(:not(vaadin-tab)) {
+  margin-left: var(--lumo-space-m);
+}
+
+/* Centered */
+
+:host([theme~='centered'][orientation='horizontal']) ::slotted(vaadin-tab:first-of-type) {
+  margin-inline-start: auto;
+}
+
+:host([theme~='centered'][orientation='horizontal']) ::slotted(vaadin-tab:last-of-type) {
+  margin-inline-end: auto;
+}
+
+/* Small */
+
+:host([theme~='small']),
+:host([theme~='small']) [part='tabs'] {
+  min-height: var(--lumo-size-m);
+}
+
+:host([theme~='small']) [part='tabs'] ::slotted(vaadin-tab) {
+  font-size: var(--lumo-font-size-s);
+}
+
+/* Minimal */
+
+:host([theme~='minimal']) {
+  box-shadow: none;
+  --_lumo-tab-marker-display: none;
+}
+
+/* Hide-scroll-buttons */
+
+:host([theme~='hide-scroll-buttons']) [part='back-button'],
+:host([theme~='hide-scroll-buttons']) [part='forward-button'] {
+  display: none;
+}
+
+/* prettier-ignore */
+:host([theme~='hide-scroll-buttons'][overflow~='start'][overflow~='end']:not([orientation='vertical'])) [part='tabs'] {
+      --_lumo-tabs-overflow-mask-image: linear-gradient(
+        90deg,
+        transparent,
+        #000 2em,
+        #000 calc(100% - 2em),
+        transparent 100%
+      );
+    }
```

Let's remove `prettier-ignore`, we don't need it as the selector fits in one line:

```suggestion
:host([theme~='hide-scroll-buttons'][overflow~='start'][overflow~='end']:not([orientation='vertical'])) [part='tabs'] {
  --_lumo-tabs-overflow-mask-image: linear-gradient(
    90deg,
    transparent,
    #000 2em,
    #000 calc(100% - 2em),
    transparent 100%
  );
}
```